lib/hive/worker.rb

```diff
@@ -212,15 +213,20 @@ def execute_job
         @log.error("  : #{e.backtrace.join("\n  : ")}")
       end
 
-      if exception
-        @job.error( exception.message )
+      if exception or File.size(@file_system.script_errors_file) > 0
```

Comment:
Should this be within `keep_script_running?` and terminate script whenever error file found, instead allowing test to complete?